tests/subyt/test_unique_filter_source.py

```
@@ -0,0 +1,148 @@
+
+    def __enter__(self):
+        return iter(self._data)
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        pass
+
+
+def test_verify_base_source():
+    fdls = FixedDataListSource()
+    with fdls as items:
+        data = [item for item in items]
```

Ruff (0.8.2)

Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)